### PR TITLE
Fix typo in documents

### PR DESCRIPTION
**Pull request by h-east**
I haven't explained each modification, but if you don't know the reason for the modification, feel free to ask me 😸

### CODE REVIEW

**Comment by Shane-XB-Qian** [comment]
correct following typo as well from #14712

```patch
diff --git a/runtime/doc/testing.txt b/runtime/doc/testing.txt
index d0f1d46c4..2e5e2a6dd 100644
--- a/runtime/doc/testing.txt
+++ b/runtime/doc/testing.txt
@@ -367,13 +367,13 @@ test_override({name}, {val})				*test_override()*
 			     memory, so that memory access errors can be found
 			     by valgrind
 		autoload     `import autoload` will load the script right
 			     away, not postponed until an item is used
 		char_avail   disable the char_avail() function
 		defcompile   all the |:def| functions in a sourced script are
-			     compiled when defined.  This is similar to using
+			     compiled when defined.  This is similar to use
 			     the |:defcompile| command in a script.
 		nfa_fail     makes the NFA regexp engine fail to force a
 			     fallback to the old engine
 		no_query_mouse  do not query the mouse position for "dec"
 				terminals
 		no_wait_return	set the "no_wait_return" flag.  Not restored
```

-- 
shane.xb.qian

**Comment by h-east** [comment]
@Shane-XB-Qian Thanks. Included in PR.

File: runtime/doc/starting.txt [hunk 1]
```diff
@@ -104,9 +104,9 @@ rvim	 vim -Z	    Like "vim", but in restricted mode (see |-Z|)   *rvim*
 rview	 vim -RZ    Like "view", but in restricted mode.	  *rview*
 rgvim	 vim -gZ    Like "gvim", but in restricted mode.	  *rgvim*
 rgview	 vim -RgZ   Like "gview", but in restricted mode.	  *rgview*
-evim     vim -y     Easy Vim: set 'insertmode' (see |-y|)	  *evim*
-eview    vim -yR    Like "evim" in read-only mode		  *eview*
```
Comment (ghgary) [inline runtime/doc/starting.txt:108]: I think your corrections to evim and eview are each missing a space after the first tab and before "vim".

**Comment by h-east** [comment]
Fix test_override()'s each {name} description in testing.txt.
I think it would be better to have "." in the last sentence of each {name} description in test_override().

File: runtime/doc/starting.txt (hunk 1 above)
Comment (h-east) [reply runtime/doc/starting.txt:108]: Thanks. I'll fix.

File: runtime/doc/syntax.txt [hunk 2]
```diff
@@ -946,7 +946,7 @@ Enables Stylus for ".astro" files. Default Value: "disable"
 NOTE: You need to install an external plugin to support stylus in astro files.
 
 
-ASPPERL and ASPVBS			*ft-aspperl-syntax* *ft-aspvbs-syntax*
+ASPPERL AND ASPVBS			*ft-aspperl-syntax* *ft-aspvbs-syntax*
```
Comment (ghgary) [inline runtime/doc/syntax.txt:951]: These two might look better on separate lines, as is done for DOCBOOK, i.e.,
`-ASPPERL and ASPVBS			*ft-aspperl-syntax* *ft-aspvbs-syntax*`
`+ASPPERL				*ft-aspperl-syntax*`
`+ASPVBS				*ft-aspvbs-syntax*`

**Comment by clason** [comment]
> @Shane-XB-Qian Thanks. Included in PR.

That is _not_ a typo, by the way; Yegappan's wording is perfectly correct English, but the "fix" is not.

File: runtime/doc/syntax.txt (hunk 2 above)
Comment (h-east) [reply runtime/doc/syntax.txt:951]: Separated.

**Comment by h-east** [comment]
Ready to merge.
(Unless there is a point or request from someone else.)

File: runtime/doc/testing.txt [hunk 3]
```diff
@@ -365,32 +365,32 @@ test_override({name}, {val})				*test_override()*
...
-			     by valgrind
+			     by valgrind.
 		autoload     `import autoload` will load the script right
-			     away, not postponed until an item is used
-		char_avail   disable the char_avail() function
+			     away, not postponed until an item is used.
+		char_avail   disable the char_avail() function.
 		defcompile   all the |:def| functions in a sourced script are
-			     compiled when defined.  This is similar to using
+			     compiled when defined.  This is similar to use
```
Comment (clason) [inline runtime/doc/testing.txt]:
```suggestion
			     compiled when defined.  This is similar to using
```

**Comment by Shane-XB-Qian** [comment]
> > @Shane-XB-Qian Thanks. Included in PR.
> 
> That is _not_ a typo, by the way; Yegappan's wording is perfectly correct English, but the "fix" is not.

@yegappan were you purposed to use `use` or `using` ?
not a matter, but my first direct feeling was the first one. thanks if you were interested to answer, or ignore.

@clason could you leave me alone? you donot need to waste your valuable time on me again, thank you very much!

**Comment by clason** [comment]
This has nothing to do with you; I care about Vim, same as anybody else here. I don't see why I should have less right pointing out mistakes than you.

**Comment by yegappan** [comment]
> > 
> > 
> > That is _not_ a typo, by the way; Yegappan's wording is perfectly correct English, but the "fix" is not.
> 
> @yegappan were you purposed to use `use` or `using` ? not a matter, but my first direct feeling was
> the first one. thanks if you were interested to answer, or ignore.

The correct word to use in that sentence is "using".

Regards,
Yegappan

**Comment by Shane-XB-Qian** [comment]
> This has nothing to do with you; I care about Vim, same as anybody else here.

interesting, you really cared official vim or neovim?

-- 
shane.xb.qian

**Comment by Shane-XB-Qian** [comment]
> > > That is _not_ a typo, by the way; Yegappan's wording is perfectly correct English, but the "fix" is not.
> > 
> > @yegappan were you purposed to use `use` or `using` ? not a matter, but my first direct feeling was
> > the first one. thanks if you were interested to answer, or ignore.
> 
> The correct word to use in that sentence is "using".

ok, thanks, my direct feeling was it should be a verb, or whatever if it is used to use that, thanks your answer.

-- 
shane.xb.qian

**Comment by julio-b** [comment]
@h-east could you please include this in your PR?

```diff
diff --git a/runtime/doc/terminal.txt b/runtime/doc/terminal.txt
index c99b882cf..f0f57b0f8 100644
--- a/runtime/doc/terminal.txt
+++ b/runtime/doc/terminal.txt
@@ -194,7 +194,7 @@ Command syntax ~
                        If [command] is provided run it as a job and connect
                        the input and output to the terminal.
                        If [command] is not given the 'shell' option is used.
-                       if [command] is NONE no job is started, the pty of the
+                       If [command] is NONE no job is started, the pty of the
                        terminal can be used by a command like gdb.
 
                        If [command] is missing the default behavior is to
```

**Comment by errael** [comment]
>  ...it should be a verb, or whatever if it is used to use that...

The phrase `similar to using` is tricky and shows some English weirdness. In particular the word `to` is closely associated with `similar` and not `using`. The phrase `similar to` means `resembles`; after `similar to` there is typically a noun. And `using` is a gerund; a gerund is: `a form that is derived from a verb but that functions as a noun, in English ending in -ing`.

**Comment by Shane-XB-Qian** [comment]
> >  ...it should be a verb, or whatever if it is used to use that...
> 
> The phrase `similar to using` is tricky and shows some English weirdness. In particular the word `to` is closely associated with `similar` and not `using`. The phrase `similar to` means `resembles`; after `similar to` there is typically a noun. And `using` is a gerund; a gerund is: `a form that is derived from a verb but that functions as a noun, in English ending in -ing`.

the difference is:
- this is similar{,} to use...
- this is (similar to) using...

thanks for your clarification, and @yegappan 's answer.
// i may more understand or perfer the first one, and used to it.

-- 
shane.xb.qian

**Comment by vim-ml** [comment]
On 2024-05-04, Shane-XB-Qian wrote:
> > > ...it should be a verb, or whatever if it is used to use that...
> >
> > The phrase `similar to using` is tricky and shows some English weirdness. In
> particular the word `to` is closely associated with `similar` and not `using`.
> The phrase `similar to` means `resembles`; after `similar to` there is
> typically a noun. And `using` is a gerund; a gerund is: `a form that is derived
> from a verb but that functions as a noun, in English ending in -ing`.
> 
> the difference is:
> - this is similar{,} to use...
> - this is (similar to) using...
> 
> thanks for your clarification, and @yegappan 's answer.
> // i may more understand or perfer the first one, and used to it.

But in the sentence in question, the first one is not an
alternative--it is improper English grammar.

Regards,
Gary

**Comment by Shane-XB-Qian** [comment]
> > > > ...it should be a verb, or whatever if it is used to use that...
> > >
> > > The phrase `similar to using` is tricky and shows some English weirdness. In
> > particular the word `to` is closely associated with `similar` and not `using`.
> > The phrase `similar to` means `resembles`; after `similar to` there is
> > typically a noun. And `using` is a gerund; a gerund is: `a form that is derived
> > from a verb but that functions as a noun, in English ending in -ing`.
> > 
> > the difference is:
> > - this is similar{,} to use...
> > - this is (similar to) using...
> > 
> > thanks for your clarification, and @yegappan 's answer.
> > // i may more understand or perfer the first one, and used to it.
> 
> But in the sentence in question, the first one is not an
> alternative--it is improper English grammar.

yea, i see, as a complete sentence, it should be the 2ed one.
// the 1st one looked was not a formal english, but strangely, that's more matching my direct feeling.

-- 
shane.xb.qian

**Comment by h-east** [comment]
Sorry, no further corrections will be made in this PR.

**Comment by chrisbra** [comment]
thanks!

**Pull request closed without merging**